### PR TITLE
optimize var: make it meet the coding conventions of golang

### PR DESCRIPTION
**Pull request by liangwenguo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kube-controller-manager/app/core.go [hunk 1]
```diff
@@ -79,8 +79,10 @@ func startServiceController(ctx ControllerContext) (http.Handler, bool, error) {
 }
 
 func startNodeIpamController(ctx ControllerContext) (http.Handler, bool, error) {
-	var clusterCIDR *net.IPNet = nil
-	var serviceCIDR *net.IPNet = nil
+	var (
+		clusterCIDR *net.IPNet = nil
```
Comment (dougm) [inline cmd/kube-controller-manager/app/core.go]: While you're at it, no need to assign `= nil` here as it is the default value, right?

Comment (nikhita) [reply cmd/kube-controller-manager/app/core.go]: +1.

@liangwenguo thanks for the clean up!

**Pull request closed without merging**